src/components/TextWithHighlight/TextWithHighlight.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "../../styles/themes";
+@import "../../styles/typography";
+
+.textWithHighlightWrapper {
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+}
+
+.textWithHighlightWrapper em.highlightText {
+  @include theme-prop(background-color, primary-selected-color);
+  @include theme-prop(color, primary-text-color);
+  border-radius: 2px;
+}
+
+.textWithHighlightWrapper.withEllipsis {
```

Comment:
no & in the script? I am not sure it's a problem but it can create a code which is less to edit is some cases, no?

Reply:
Sometimes it's creating a really messy files... But @orrgottlieb wanted css files to be fully unwrapped. 
Also this functionality is implemented by independent plugin and it will require some time investments to modify this process, if we want to. Alternative solution would be to manually edit the especially messy files
Waiting for @orrgottlieb with this one

Reply:
Orr decision was to keep everything as it is, but to manually refactor messy stylesheets